server/src/main/java/org/apache/cloudstack/consoleproxy/ConsoleAccessManagerImpl.java

```diff
@@ -66,6 +70,9 @@
 import java.util.List;
 import java.util.Map;
 import java.util.UUID;
+import java.util.concurrent.Executors;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.TimeUnit;
 
 public class ConsoleAccessManagerImpl extends ManagerBase implements ConsoleAccessManager {
```

Comment:
@nvazquez 
this should implement `Configurable` class and its methods as well

Reply:
Done, thanks